examples/tutorials/streamwriter_basic_tutorial.py

```diff
@@ -0,0 +1,674 @@
+#
+
+# 3. Write the data
+with s.open():
+    s.write_audio_chunk(0, WAVEFORM)
+
+######################################################################
+#
+
+Audio(path)
+
+######################################################################
+#
+# Now we look into each step in more detail.
+
+######################################################################
+#
+# Write destination
+# -----------------
+#
+# StreamWriter supports different types of write destinations
+#
+# 1. Local files
+# 2. File-like objects
+# 3. Streaming protocols (such as RTMP and UDP)
+# 4. Media devices (speakers and video players) †
+#
+# † For media devices, please refer to
+# `StreamWriter Advanced Usages <./streamwriter_advanced.html>`__.
+#
+
+######################################################################
+# Local files
+# ~~~~~~~~~~~
+#
+# StreamWriter supports saving media to local files.
+#
+#
+# .. code::
+#
+#    StreamWriter(dst="audio.wav")
+#
+#    StreamWriter(dst="audio.mp3")
+#
+# This works for still images and videos as well.
+#
+# .. code::
+#
+#    StreamWriter(dst="image.jpeg")
+#
+#    StreamWriter(dst="video.mpeg")
+#
+
+######################################################################
+# File-like objects
+# ~~~~~~~~~~~~~~~~~
+#
+# You can also pass a file-lie object. A file-like object must implement
+# ``write`` method conforming to :py:attr:`io.RawIOBase.write`.
+#
+# .. code::
+#
+#    # Open the local file as fileobj
+#    with open("audio.wav", "wb") as dst:
+#        StreamWriter(dst=dst)
+#
+# .. code::
+#
+#    # In-memory encoding
+#    buffer = io.BytesIO()
+#    StreamWriter(dst=buffer)
+#
+
+######################################################################
+# Streaming protocols
+# ~~~~~~~~~~~~~~~~~~~
+#
+# You can stream the media with streaming protocols
+#
+# .. code::
+#
+#    # Real-Time Messaging Protocol
+#    StreamWriter(dst="rtmp://localhost:1234/live/app", format="flv")
+#
+#    # UDP
+#    StreamWriter(dst="udp://localhost:48550", format="mpegts")
+#
+
+######################################################################
+#
+# Configuring output streams
+# --------------------------
+#
+# Once the destination is specified, the next step is to configure the streams.
+# For typical audio and still image cases, only one stream is required,
+# but for video with audio, at least two streams (one for audio and the other
+# for video) need to be configured.
+#
+
+######################################################################
+# Audio Stream
+# ~~~~~~~~~~~~
+#
+# An audio stream can be added with
+# :py:meth:`~torchaudio.io.StreamWriter.add_audio_stream` method.
+#
+# For writing regular audio files, at minimum ``sample_rate`` and ``num_channels``
+# are required.
+#
+# .. code::
+#
+#    s = StreamWriter("audio.wav")
+#    s.add_audio_stream(sample_rate=8000, num_channels=2)
+#
+# If the above case, the encoding pipeline expects a tensor of ``torch.float32`` and
```

Comment:
looks good. could you also mention that ``torch.float32`` is the default format of the audio stream?